.circleci/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM centos:7
+
+RUN yum update -y
+
+RUN  yum install -y \
+        https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+        git \
+        yum-utils \
+        centos-release-scl
+
+RUN yum install -y \
+        devtoolset-8 \
+        java-1.8.0-openjdk \
```

Comment:
We use oracle's jdk and jre for docker deployment.
I believe that in long term we should use the same JDK/JRE in both deployments.
I think adding a TODO about this is enough for now, e.g. you will not be bothered with this detail at the moment.

Reply:
i think actually i only need the JRE here and it is used to run bazel. there is no need to couple this JRE to the one we use in production deployment.